cypher-shell/src/main/java/org/neo4j/shell/Main.java

```diff
@@ -84,6 +82,15 @@ void startShell(@Nonnull CliArgs cliArgs) {
         }
     }
 
+    private Logger instantiateLogger(@Nonnull CliArgs cliArgs) {
+        Logger logger = new AnsiLogger(cliArgs.getDebugMode());
+        logger.setFormat(cliArgs.getFormat());
+        if (cliArgs.isStringShell()) {
```

Comment:
I think it is necessary to make this:

```
if (cliArgs.isStringShell() && AUTO.equals(cliArgs.getFormat())) {
```

otherwise a user can specify `--format=verbose` and that would be ignored.